### DISCUSSION TITLE
Nitrate

### DISCUSSION THREAD

**Discussion by soerenthomsen** [GliderToolsCommunity/GliderTools#79] [Ideas]
Dear glider folks, 

Gliders can also be used to observe nitrate at high spatial and temporal resolution. In 2014 we deployed one off Mauritania "Remote and local drivers of oxygen and nitrate variability in the shallow oxygen minimum zone off Mauritania in June 2014 https://bg.copernicus.org/articles/16/979/2019/). 

Are there people out here working on nitrate measurements on gliders? i.e. with a SUNA?

**Comment by bastienqueste** [comment]
This is one where I have nothing to contribute but I'm very eager to see the Best Practices :)
Planned deployments next year in the Amundsen Sea.

**Reply by soerenthomsen** [reply]
You have a SUNA on the SeaExplorers?

**Reply by bastienqueste** [reply]
Absolutely. Fully integrated.
I've not deployed one myself, but it has been deployed with FRRF sensors in the past ( https://pbs.twimg.com/media/EUhUGlIWAAIAKV5.jpg ).

We bought a SUNA for Seagliders and then gave up. SX much more promising. G3 probably most developed though.

**Comment by tomhull** [comment]
I've been working with NOC LoC Nitrate (and Phosphate) instruments on seagliders. The integration is getting there, but it's not at the point where the values are present in the basestation netcdf files.

https://doi.org/10.1016/j.marchem.2018.07.005
